### ISSUE TITLE
`Uninstall-Font`: Add support for ignoring unregistered fonts

### ISSUE THREAD

**Issue by felipecrs** [ralish/PSWinGlue#7]
On scripts scenarios, some times we need to ensure something is simply not installed.

For example:

```console
❯ sudo apt remove -y jq
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package 'jq' is not installed, so not removed
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
❯ echo $?
0
```

- My use case the same as mentioned in https://github.com/ralish/PSWinGlue/issues/6, which is to ensure there are no versions of the font installed before issuing the installation of it (to avoid having multiple versions installed at the same time).

- While I would prefer that `Install-Font.ps1` simply had a `-UninstallFirst` flag, I believe it would also be useful for `Uninstall-Font.ps1` to have a `-IgnoreMissing` flag.

**Comment by ralish** [comment]
@felipecrs I should be able to take a look at this on the coming weekend.

**Comment by ralish** [comment]
Done and released in `v0.6.5`. The `Uninstall-Font` command now has an `-IgnoreNotPresent` parameter.
